src/network/axiosClient.js

```diff
@@ -1,21 +1,23 @@
-import axios from "axios";
-import { toast } from "react-toastify";
+import axios from 'axios';
+import { toast } from 'react-toastify';
 
 const axiosClient = axios.create();
 
 axiosClient.defaults.baseURL = `${import.meta.env.VITE_BASE_URL}`;
 
-axiosClient.interceptors.response.use( response => {
-  return response;
-}, error => {
-  if (error.response.status === 500) {
-    toast.error("Something went wrong, please try again!");
-  } 
-  return Promise.reject(error);
-});
+axiosClient.interceptors.response.use(
+  response => {
+    return response;
+  },
+  error => {
+    if (error?.response?.status >= 500 && error?.response?.status < 600) {
```

Comment:
Define constants for status codes because we might need them at multiple places. Secondly, checking that status code starting with 5 would be enough for this case.